static/app/components/jsonBlock.tsx

```diff
@@ -0,0 +1,31 @@
+import {useMemo} from 'react';
+import {chromeDark, chromeLight, ObjectInspector} from 'react-inspector';
+
+import ConfigStore from 'sentry/stores/configStore';
+
+type Props = {
+  data: any;
```

Comment:
isn't there a better type we can add here?

Reply:
Yeah, we shouldn't be explicitly allowing `any` here. Instead it should be whatever the wrapped component allows.